validator/validator-main.protoascii

```diff
@@ -3145,6 +3145,25 @@ tags: {
     }
   }
 }
+# AMP IMA VIDEO
+tags: {
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-ima-video > script[type=application/json]"
+  unique: true
+  mandatory_parent: "AMP-IMA-VIDEO"
+  attrs: {
```

Comment:
Let's add a dispatch key attribute. This will look like:

attrs: {
  name: "amp-ima-video-config"
  value: ""
  dispatch_key: NAME_DISPATCH
}

This will force the validator to only consider this tagspec for matching if that attribute is present on the tag it's matching. Therefore this tagspec won't produce errors for generic `<script>` tags found in the document.

Users will need to add that attribute to their script tag, if that's OK.

In the future, we can improve the validator matching and relax this requirement.